Makefile

```diff
@@ -56,6 +57,7 @@ build-tests: build-extended-test
 .PHONY: build-tests
 
 build-extended-test:
+	go generate ./test/extended/ 
```

Comment:
This is covered by hack/update-generated-bindata.sh, you should be calling that.  And we generally don't call update bindata automatically.

Reply:
it took me more than a few minutes to figure out the 'test not found' panic after adding a new test/extended/file.  I didn't see any documentation in the test/extended/README about generating `test/extended/util/annotate/generated/zz_generated.annotations.go`.  I can add that documentation instead, then.

Reply:
I know it's also in hack/update-generated-bindata.sh, but that script also fails unless I add 
`export GO111MODULE=off`